server/events/unlock_command_runner.go

```diff
@@ -11,8 +12,7 @@ func NewUnlockCommandRunner(
 	vcsClient vcs.Client,
 ) *UnlockCommandRunner {
 	return &UnlockCommandRunner{
-		deleteLockCommand: deleteLockCommand,
-		vcsClient:         vcsClient,
+		vcsClient: vcsClient,
```

Comment:
why did you remove `deleteLockCommand` attribute?

Reply:
Oops must have slipped 😅 I've reverted this change